contrib/kafka/cmd/dispatcher/main.go

```diff
@@ -34,6 +35,12 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 )
 
+const (
+	// These are Environment variable names.
+	kafkaTLSENnabled = "KAFKA_TLS_ENABLE"
```

Comment:
Define these in pkg somewhere, so they can be shared between the two?

Reply:
:+1: